routes/orders.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const router = express.Router();
+const ordersCtrl = require('../controllers/orders');
+
+// Create a new order (Add item to cart)
+router.post('/orders', ordersCtrl.create);
```

Comment:
you can re-write this route to say: `router.post('/:productId', ordersCtrl.create);`

you can update the comment to say // POST /orders/:productId -- add item to cart

because that's the route it'll be directed to once we click to add an item to cart.

we will be adding items from a product detail page (add to cart button), not the order/cart page.

once this route is updated, you can modify `ordersCtrl.create`. I'll leave a comment on the function.

Reply:
Post route now reads: ```router.post('/:productId', ordersCtrl.create);``` and comment has been updated